airflow/models/dag.py

```diff
@@ -778,16 +785,20 @@ def set_dependency(self, upstream_task_id, downstream_task_id):
     def get_task_instances(
             self, start_date=None, end_date=None, state=None, session=None):
         if not start_date:
+            # TODO why 30?
```

Comment:
PLEASE do not add todo comments in the code. Either find out why and fix it or document it, or don't add the comment. This doesn't help anyone.